lib/puppet/module_tool/errors/shared.rb

```diff
@@ -136,17 +137,17 @@ def initialize(options)
       @module_name = options[:module_name]
       @modules     = options[:installed_modules]
       @action      = options[:action]
-      super "Could not #{@action} '#{@module_name}'; module appears in multiple places in the module path"
+      super _("Could not %{action} '%{module_name}'; module appears in multiple places in the module path") % { action: @action, module_name: @module_name }
```

Comment:
could you add a translator note that `module path` here refers to a set of directories that puppet knows of to install modules in?